feelpp/feel/feelfilters/detail/meshcontiguousnumberingmapping.hpp

```diff
@@ -39,10 +39,11 @@ struct MeshContiguousNumberingMapping
     using range_element_type = elements_reference_wrapper_t<mesh_type>;
     using point_ref_type = boost::reference_wrapper< typename mesh_type::point_type const>;
 
-    explicit MeshContiguousNumberingMapping( mesh_type* mesh, bool interprocessPointAreDuplicated = false )
+    explicit MeshContiguousNumberingMapping( mesh_type* mesh, bool interprocessPointAreDuplicated = false, bool exportByParts = true )
```

Comment:
@prudhomm The idea was to put a struct here (not just a boolean), with possibility to define a more complex behavior.
For example, A part name toto will contain a range of elements, another part named titi with another range, …
This is required for our kub application.
For a simpler use, A `JSON` object and a mesh can set up easily this struct.

Reply:
is it possible to do that in another specific branch/PR ? I guess there would be more development than that.
this will unbreak the exports

Reply:
Yes, I understand. I only thought of adding the structure with a boolean and init from a Mesh. This will avoid breaking the API in the future.